docs/changes.rst

```diff
@@ -7,6 +7,8 @@ This chapter describes the changes coming with each new version of pygount.
 
 Version 1.6.1, 2023-07-xx
 
+* Fixed missing check for seekable file handles (issue
+  `#114 <https://github.com/roskakori/pygount/issues/114>`_).
 * Fix the ReadTheDocs documentation build by switching to the built-in
```

Comment:
My bad, but please clean it up: Fix -> Fixed (past tense, unlike the commit comments;  maybe we should unify the wording of changes.rst  with the commit comments at some point).

Reply:
Fixed: [0888a5e](https://github.com/roskakori/pygount/pull/115/commits/0888a5e0dba4b1d58a64f5c9073a51be1a337669)